docs/source/module-pysages-methods.rst

```diff
@@ -11,6 +11,7 @@ Available Methods
    pysages.methods.ann.ANN
    pysages.methods.ffs.FFS
    pysages.methods.funn.FUNN
+   pysages.methods.metad.Metadynamics
    pysages.methods.umbrella_integration.UmbrellaIntegration
    pysages.methods.harmonic_bias.HarmonicBias
 
```

Comment:
Could you also add `pysages.methods.utils.MetaDLogger` under,

.. autosummary::

   pysages.methods.utils.HistogramLogger
   **pysages.methods.utils.MetaDLogger**

Reply:
sure, done.